redbot/core/utils/antispam.py

```diff
@@ -45,14 +47,16 @@ def __interval_check(self, interval: AntiSpamInterval):
     @property
     def spammy(self):
         """
-        use this to check if any interval criteria are met
+        Whether any antispam intervals are active.
         """
         return any(self.__interval_check(x) for x in self.__intervals)
 
     def stamp(self):
         """
-        Use this to mark an event that counts against the intervals
-        as happening now
+        Mark against the list of antispam intervals.
```

Comment:
Maybe this should just say that we're recording an event (or event timestamp)? It probably wouldn't hurt to mention that it records this event as something that happened *now*.